env

```diff
@@ -0,0 +1,2 @@
+DATABASE_URL=sqlite:///app.db
```

Comment:
I think your env file needs to be named ".env". Without the "." in front, it just shows a regular file and gets added like a regular file with all the secret stuff. (I'm assuming it's here forever now, lol. It's fine though because, I believe, the plan is to put the DB on AWS or something).

Reply:
I should add that .gitignore should pick the file up if it's renamed to ".env".